src/Workspaces/CSharp/Portable/CodeGeneration/EventGenerator.cs

```diff
@@ -199,6 +200,14 @@ private static BlockSyntax GenerateBlock(IMethodSymbol accessor)
                         tokens.Add(SyntaxFactory.Token(SyntaxKind.StaticKeyword));
                     }
 
+                    // An event is readonly if its accessors are readonly.
```

Comment:
What do we do in the case that we can't convert it to the `readonly` keyword?

Do we show nothing or do we still display the attribute on the annotated members even though it isn't valid C#?

Reply:
I'm not sure what you mean by convert it to the 'readonly' keyword. The attributes get filtered out so they won't show up in the C# `PEMethodSymbol.GetAttributes()` and therefore won't show up in the C# MetadataAsSource.